api/config/v1alpha1/shared_types.go

```diff
@@ -8,14 +8,16 @@ package v1alpha1
 import corev1 "k8s.io/api/core/v1"
 
 const (
-	// DefaultEnvoyReplicas is the default number of Envoy replicas.
-	DefaultEnvoyReplicas = 1
+	// DefaultDeploymentReplicas is the default number of deployment replicas.
+	DefaultDeploymentReplicas = 1
 	// DefaultDeploymentCPUResourceRequests for deployment cpu resource
 	DefaultDeploymentCPUResourceRequests = "100m"
 	// DefaultDeploymentMemoryResourceRequests for deployment memory resource
 	DefaultDeploymentMemoryResourceRequests = "512Mi"
 	// DefaultEnvoyProxyImage is the default image used by envoyproxy
 	DefaultEnvoyProxyImage = "envoyproxy/envoy-dev:latest"
+	// DefaultRateLimitImage is the default image used by ratelimit.
+	DefaultRateLimitImage = "envoyproxy/ratelimit:f28024e3"
```

Comment:
I kind of feel like either this shouldn't be pinned, or the Envoy version should be -- having one pinned to a specific version and the other not feels weird to me. How will this get updated?

Reply:
ref  ratelimit_deployment.go